plugins/Monstro/Monstro.cpp

```diff
@@ -316,32 +316,32 @@ void MonstroSynth::renderOutput( fpp_t _frames, sampleFrame * _buf  )
 	// declare working variables for for loop
 
 	// phase manipulation vars - these can be reused by all oscs
-	float leftph;
-	float rightph;
-	float pd_l;
-	float pd_r;
+	float leftph = 0.0f;
+	float rightph = 0.0f;
+	float pd_l = 0.0f;
+	float pd_r = 0.0f;
```

Comment:
There's two below this that are using `name(value)`, let's change this to `=` instead.